### DISCUSSION TITLE
Python: Query data into dataframe

### DISCUSSION THREAD

**Discussion by ben112233** [sqlitecloud/discussions#10] [Feature request] [answered]
I am working on a database project using Python. I have tried to query data from the database and the queried output is stored inside the SqliteCloudResultSet which I need to turn into a dataframe manually to view the queried result easily. I am wondering whether these features can be added. Thank you!

**Comment by danielebriggi** [comment, marked as answer]
Yes! In the next days we are going to release an update to the `sqlitecloud` package.
It will allow you to use the connection to SQLite Cloud directly into the Pandas dataframe like:

```
import pandas as pd
import sqlitecloud

con = sqlitecloud.connect("sqlitecloud://myhost.sqlitecloud.io/dbname?apikey=abc1234")
df = pd.read_sql_query("SELECT * from albums", conn)

print(df.head())

conn.close()
```

Is this solution the feature you are you looking for?

**Reply by ben112233** [reply]
yes! It will be nice to have this features!

**Reply by danielebriggi** [reply]
Today we've released an update that introduces support for the feature you're interested in. You can find an example of how to use this feature in here: [sqlite-cloud-for-pandas-dataframe](https://github.com/sqlitecloud/sqlitecloud-py#sqlite-cloud-for-pandas-dataframe) and in the [samples Jupyter notebook](https://github.com/sqlitecloud/sqlitecloud-py/blob/main/samples.ipynb).

If you encounter any issues or have questions, please don't hesitate to contact us!
